cert-manager/README.md

```diff
@@ -0,0 +1,165 @@
+# cert-manager
+
+cert-manager plugin for Headlamp.
```

Comment:
Can you please add a bit more here? So people new to cert-manager will have more of an idea. Also what this plugin does for users evaluating the plugin.

- A screenshot or your video would be helpful too.
- What about adding a section for testing this plugin from the PR description?

@vyncent-t I know you have some good ideas on plugin README improvements... can you please review this README?

Reply:
The README has been updated with those suggestions.